skymp5-server/cpp/unit/DistContentsTest.h

```diff
@@ -0,0 +1,217 @@
+#include "TestUtils.hpp"
+#include <catch2/catch.hpp>
+
+namespace {
+size_t PrintMissing(const std::set<std::filesystem::path>& whatIsMissing,
+                    const std::set<std::filesystem::path>& whereIsMissing,
+                    const char* whatIsMissingName,
+                    const char* whereIsMissingName)
+{
+  std::vector<std::filesystem::path> missing;
+  for (auto& p : whatIsMissing) {
+    if (whereIsMissing.count(p) == 0) {
+      missing.push_back(p);
+    }
+  }
+  if (!missing.empty()) {
+    std::cout << "Missing some elements of '" << whatIsMissingName << "' in '"
+              << whereIsMissingName << "':\n";
+    for (auto& p : missing) {
+      std::cout << " - " << p << "\n";
+    }
+  }
+  return missing.size();
+}
+}
+
+TEST_CASE("Distribution folder must contain all requested files",
+          "[DistContents]")
+{
+  auto distDir = std::filesystem::u8path(DIST_DIR);
+  auto begin = std::filesystem::recursive_directory_iterator(distDir);
+  auto end = std::filesystem::recursive_directory_iterator();
+
+  std::set<std::filesystem::path> paths;
+
+  for (auto it = begin; it != end; ++it) {
+    if (!it->is_directory()) {
+      auto& path = it->path();
+      paths.insert(std::filesystem::relative(path, distDir));
+    }
+  }
+
+  const std::set<std::filesystem::path> expectedPaths = {
+    "client\\Data\\Platform\\Distribution\\CEF\\cef.pak",
```

Comment:
Maybe replace `\` to `/`? This will also make test more portable, as it will support Linux too.

BTW I think we should make it a text file. With `std::filesystem` we can get parent path for `__FILE__` and concatenate it with some kind of `static` dir and file inside of it.